getting-started/config-supply-chain.md

```diff
@@ -1,61 +1,57 @@
+2. Create a `values.yaml` file, and install the Supply Chain Security Tools - Policy Controller. For instructions, see [Install Supply Chain Security Tools - Policy Controller](../scst-policy/install-scst-policy.md).
 
-3. Configure a `ClusterImagePolicy` resource to verify image signatures when deploying resources. The resource must be named `image-policy`.
+3. Configure at least one `ClusterImagePolicy` resource to verify image signatures when deploying resources. For instructions, see [Create a `ClusterImagePolicy` resource](../scst-policy/configuring.md#create-cip-resource).
 
     For example:
 
     ```yaml
     ---
-    apiVersion: signing.apps.tanzu.vmware.com/v1beta1
+    apiVersion: policy.sigstore.dev/v1beta1
     kind: ClusterImagePolicy
     metadata:
-       name: image-policy
+      name: example-policy
     spec:
-       verification:
-         exclude:
-           resources
-             namespaces:
-             - kube-system
-             - test-namespace
-             - <TAP system namespaces>
-         keys:
-         - name: first-key
-           publicKey: |
-             -----BEGIN PUBLIC KEY-----
-             <content ...>
-             -----END PUBLIC KEY-----
-         images:
-         - namePattern: registry.example.org/myproject/*
-           keys:
-           - name: first-key
-
+      images:
+      - glob: registry.example.org/myproject/*
+      authorities:
+      - key:
+          data: |
+            -----BEGIN PUBLIC KEY-----
+            <content ...>
+            -----END PUBLIC KEY-----
     ```
 
-> **Note:** System namespaces specific to your cloud provider might need to be excluded from the policy.
+4. Enable the policy controller verification in your namespace by adding the label
+`policy.sigstore.dev/include: "true"` to the namespace resource.
+    For example:
+    ```console
+    kubectl label namespace my-secure-namespace policy.sigstore.dev/include=true
+    ```
 
-To prevent the Image Policy Webhook from blocking components of Tanzu Application Platform, VMware recommends configuring exclusions for Tanzu Application Platform system namespaces listed in [Create a `ClusterImagePolicy` resource](../scst-sign/configuring.md#create-cip-resource).
+> **Note:** Supply Chain Security Tools - Policy Controller only validates resources in namespaces
+that have chosen to opt-in.
 
-When you apply the `ClusterImagePolicy` resource, your cluster requires valid signatures for all images that match the `namePattern:` you define in the configuration. For more information about configuring an image signature policy, see [Configuring Supply Chain Security Tools - Sign](../scst-sign/configuring.md).
+When you apply the `ClusterImagePolicy` resource, your cluster requires valid signatures for all images that match the `spec.images.glob[]` you define in the configuration. For more information about configuring an image policy, see [Configuring Supply Chain Security Tools - Policy](../scst-policy/configuring.md).
```

Comment:
And likewise in the subsequent usages of `Supply Chain Security Tools - Policy`

Reply:
I wrote `Supply Chain Security Tools - Policy` because that is the section name in scst-policy/configuring.md. Should I update it also in scst-policy/configuring.md or just in this doc?